bamtk.c

```diff
@@ -181,6 +182,7 @@ static void usage(FILE *fp)
 "     fasta          converts a BAM to a FASTA\n"
 "     import         Converts FASTA or FASTQ files to SAM/BAM/CRAM\n"
 "     reference      Generates a reference from aligned data\n"
+"     reset          Remove alignment changes in sequences and make them as raw reads\n"
```

Comment:
This line is longer than all the others, meaning it wraps on an 80-char terminal.  While people often have larger terminals, maybe we can rephrase it as "Reverts aligned reads to unaligned" or similar?

Reply:
Done.